tests/e2e/plugin-init.spec.ts

```diff
@@ -0,0 +1,28 @@
+import {execSync} from "child_process";
+import fs from "fs";
+
+const taqueriaProject = './e2e/auto-test';
+
+describe("E2E Testing for taqueria general functionality", () => {
+    test('Verify that taq init creates test folder', () => {
+
+        try {
+            execSync("taqueria init e2e/auto-test")
```

Comment:
This should be `taq` instead of taqueria

Reply:
Fixed.